pkg/gadgets/trace/network/types/types.go

```diff
@@ -18,16 +18,17 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	PktType string `json:"pktType,omitempty"`
-	Proto   string `json:"proto,omitempty"`
+	PktType string `json:"pktType,omitempty" column:"type,maxWidth:9"`
+	Proto   string `json:"proto,omitempty" column:"proto,maxWidth:6"`
```

Comment:
We are using `maxWidth: 5` for the `Proto` in trace/bind as it is usually only TCP/UDP.

Reply:
Maybe it is worth adding a template for it as well?

```golang
columns.MustRegisterTemplate("proto", "maxWidth:5")
```

cc @flyth